pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/dao/impl/PatientRecordDAOImpl.java

```diff
@@ -58,7 +58,6 @@ public PatientRecordDTO create(PatientRecordDTO dto) {
 		patient.setSsn(dto.getSsn());
 		patient.setGender(dto.getGender());
 		patient.setPhoneNumber(dto.getPhoneNumber());
-		patient.setOrgPatientRecordId(dto.getOrgPatientRecordId());
```

Comment:
This used to say setOrgPatientId(dto.getOrgPatientId()) and it contains the ID the an organization assigns internally to their patients. It's a String and comes back in the SOAP response - should be getting parsed and set from there.